File: backend/src/main/java/com/licenseair/backend/domainModel/ProfileModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.sql.Date;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Profile
*/
@Data()
public class ProfileModel {
  public Long id = null;

  /**
  * 是否删除
  */
  public Integer deleted = null;

  /**
  * 公司
  */
  public String company = null;

  /**
  * 昵称
  */
  public String nickname = null;

  /**
  * 用户id
  */
  public Long user_id = null;

  /**
  * 头像
  */
  public String avatar = null;

  /**
  * 生日
  */
  public Date birthday = null;

  /**
  * 简介
  */
  public String intro = null;

  /**
  * 所在城市
  */
  public String city = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/profile.model.ts
class ProfileModel {
  public id: number = 0;

  // deleted 是否删除
  public deleted: boolean = false;

  // company 公司
  public company?: string;

  // nickname 昵称
  public nickname?: string;

  // user_id 用户id
  public user_id: number = 0;

  // avatar 头像
  public avatar?: string;

  // birthday 生日
  public birthday?: string;

  // intro 简介
  public intro?: string;

  // city 所在城市
  public city?: string;

  public created_at: string;

  public updated_at: string;
}

const ProfileModelFields = {
  id: "id",
  deleted: "deleted",
  company: "company",
  nickname: "nickname",
  user_id: "user_id",
  avatar: "avatar",
  birthday: "birthday",
  intro: "intro",
  city: "city",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {ProfileModel, ProfileModelFields};
